### DISCUSSION TITLE
Codelite Problems

### DISCUSSION THREAD

**Discussion by Ethan-Bit-256** [premake/premake-core#1779] [Q&A] [answered]
Whenever I try generate Codelite files I get this error. It's not even linked it my Premake5.lua file!

Error: [string "src/base/fileconfig.lua"]:54: attempt to index a nil value (field 'configs')
Heres the required command: call ThirdParty\Premake5\Premake5.exe codelite

You can download my premake problem below.
[MyPremakeIssue.zip](https://github.com/premake/premake-core/files/7763364/MyPremakeIssue.zip)

**Comment by Ethan-Bit-256** [comment, marked as answer]
NVM, fixed it!

**Reply by KyrietS** [reply]
You can share your solution in case someone comes here with a similar problem / error message 🙃
